### PR TITLE
add basic usage README

### PR DESCRIPTION
**Pull request by nyaxt**
@jyasskin PTAL

### CODE REVIEW

File: go/signedexchange/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# gen-signedexchange
+
+This directory holds a fork of the [gen-signedexchange tool](https://github.com/WICG/webpackage/tree/master/go/signedexchange) that best matches Chrome implementation of now.
```
Comment (jyasskin) [inline go/signedexchange/README.md]: "best matches Chrome implementation of now" -> "best matches the current Chrome implementation"?

File: go/signedexchange/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+## Install
+Simply go get:
+```
+go get github.com/nyaxt/webpackage/go/signedexchange/cmd/gen-{certurl,signedexchange}
+```
+
+## Basic Usage
+Suppose you want to create a signed exchange envelope `foo.sxg` which encapsulates https://example.com/article.html.
+
+First, prepare a private key `key.pem` and its X509 certificate `cert.pem` for dthe domain example.com encoded in PEM format. For now, the private key must be a RSA 2048 bits private key.
```
Comment (jyasskin) [inline go/signedexchange/README.md]: sp: dthe

Comment (jyasskin) [inline go/signedexchange/README.md]: s/2048 bits/2048-bit/

File: go/signedexchange/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+First, convert the certificate to certUrl format using gen-certurl:
+```
+gen-certurl cert.pem > cert.pem.msg
+```
+
+Then, host the certUrl at public URL. In this example, let's suppose we hosted the cert.pem.msg at https://cert.example.org/cert.pem.msg.
```
Comment (jyasskin) [inline go/signedexchange/README.md]: s/at public URL/at a public URL/

**Review by jyasskin — APPROVED** [review]
Thanks!

File: go/signedexchange/README.md (hunk 2 above)
Comment (twifkak) [inline go/signedexchange/README.md]: also SHA-256 signing algorithm

**Pull request merged**